docs/smac_mod/helpx.txt

```diff
@@ -42,17 +40,14 @@ Terraform
 ^Increases Probe Team base morale.
 ^Increases COMMERCE income from Treaties and Pacts.
 ^Entire planetary map is revealed.
-^Genetic Warfare atrocity can be committed by probe teams.
+^Genetic Warfare atrocity can be commited by probe teams.
```

Comment:
commitTed seems to be right though... https://www.thefreedictionary.com/Committed

Reply:
Oops, that's a typo from the original helpx file. I copied and pasted without fixing it, my mistake.